### PR TITLE
[Web Portal] home page

### PR DESCRIPTION
**Pull request by sunqinzheng**
#2513

### CODE REVIEW

**Comment by sunqinzheng** [comment]
图片

File: src/webportal/src/app/home/index.jsx [hunk 1]
```diff
@@ -0,0 +1,259 @@
...
+          )}
+          <div className={c(t.center, t.mt3)}>
+            <TextField componentRef={usernameRef} label='Username'/>
+          </div>
+          <div className={c(t.center, t.mt3)}>
+            <TextField
+              componentRef={passwordRef}
+              label='Password'
+              type='password'
+              onKeyPress={(e) => {
```
Comment (Gerhut) [inline src/webportal/src/app/home/index.jsx]: Please use a form with onSubmit instead

File: src/webportal/src/app/home/index.jsx [hunk 2]
```diff
@@ -0,0 +1,259 @@
...
+          Platform for AI
+        </div>
+        <div className={c(FontClassNames.large, t.white, t.dim, t.pointer)} onClick={() => setLoginModal(true)}>
+          Sign in
+        </div>
+      </div>
+      {/* content */}
+      <div className={c(t.flexAuto, t.flex, t.flexColumn, t.relative)}>
+        {/* jumbotron */}
+        <Jumbotron showLoginModal={() => setLoginModal(true)} />
```
Comment (Gerhut) [inline src/webportal/src/app/home/index.jsx]: It's better to wrap the callback with `useCallback` to cache it

**Comment by Gerhut** [comment]
Also, please resolve merge conflicts

File: src/webportal/src/app/home/index.jsx (hunk 2 above)
Comment (sunqinzheng) [reply src/webportal/src/app/home/index.jsx]: fixed

File: src/webportal/src/app/home/index.jsx (hunk 1 above)
Comment (sunqinzheng) [reply src/webportal/src/app/home/index.jsx]: fixed

**Pull request merged**